packages/server/src/environment.js

```diff
@@ -78,6 +78,7 @@ module.exports = {
   ALLOW_DEV_AUTOMATIONS: process.env.ALLOW_DEV_AUTOMATIONS,
   DISABLE_THREADING: process.env.DISABLE_THREADING,
   SQL_MAX_ROWS: process.env.SQL_MAX_ROWS,
+  TEST_ENV: process.env.TEST_ENV,
```

Comment:
It might be nice if we can create a more multi purpose flag so that we can do similar changes in other environments. e.g. a var called `DEPLOY_ENV`  with the value `qa`, this could be extended to `dev`, `staging` if we wanted to use custom templates there in future.

The usage could then look more like the below, so that we don't need to add a new env var each time. 
```js
let type = "app"
if (env.DEPLOY_ENV === "qa") {
   type = "qa"
}
if (env.DEPLOY_ENV === "dev") {
   type = "dev"
}
```

Another option would be to just go very specific and use a variable `TEMPLATE_REPOSITORY` that defaults to `app` when not set and it's value would be used directly in determining the type. Both would work, and the current solution is equally fine, I'm just not a fan of introducing a single flag for indicating if it is test env or not.

Reply:
Yea totally agree, that's a good point! Went with the latter option, which I think is a bit nicer than having to add a new conditional for every environment.